### DISCUSSION TITLE
Can I use the Bevy Logo as a Spashscreen in a game

### DISCUSSION THREAD

**Discussion by rowanfr** [bevyengine/bevy#10861] [Q&A] [answered]
I want to know if I can use the Bevy logo as a splash screen before my game is fully loaded similar to Unity so that I can credit the Bevy game engine in the game immediately. Would this be a violation of copyright? Unfortunately I don't know who owns the rights to the Bevy logo or under what conditions it can be used.

**Comment by Gingeh** [comment, marked as answer]
Cart has said on discord:
> yes you can use the bevy logo for splash screens, and if you have modified it run it by me first

https://discord.com/channels/691052431525675048/691052431974465548/1107774150740807763

**Comment by rlidwka** [comment]
Here is an interesting issue I've just found while researching this: https://github.com/bevyengine/bevy_github_ci_template/issues/45

> I retain all rights to the logo atm. The license section explicitly calls out that the code is dual licensed under mit/apache.
> https://discord.com/channels/691052431525675048/745355529777315850/995411630563262519

--------------

This summer, there was a huge controversy regarding Rust trademark policy. It even lead to a fork, where this lovely discussion is from: https://github.com/crablang/crab/issues/55#issuecomment-1567584547

In bevy, as far as I can find, this situation is even worse. Bevy logo doesn't seem to have any standardized license at all.

Yes, you are explicitly allowed to use bevy's logo in splash screens, with a few caveats:
 - if you believe that discord message carries any legal weight it all
 - if you don't make any modifications (rust trademark policy at least allows resizing, but with bevy you'd have to ask Cart)
 - nobody said that you can redistribute it (so copying bevy logo to your opensource repository might be questionable)
 - nobody said this permission is irrevocable (so Cart can ask you to remove it at any time)

I believe open-source projects should have their logo licensed like this:
https://github.com/godotengine/godot/blob/master/LOGO_LICENSE.txt

But with current Bevy's policies, I'd avoid using its logo if I can help it, because I feel like someone is going to get the removal request eventually. And just imagine the consequences of being asked to remove bevy logo from already released steam game, if that game turns out to be slightly controversial...
